### DISCUSSION TITLE
Showcase: Install C website got 1200 clicks from Google this month

### DISCUSSION THREAD

**Discussion by Welding-Torch** [imfing/hextra#461] [Show and tell]
https://installc.org/ uses Hextra. I developed it last year around the same time I opened a few issues on this repo when I was getting stuck. Thank you so much for providing Hextra to make a beautiful website for this project of mine ❤️

**Comment by imfing** [comment]
do you mind send a pull request to add it to the showcase page?

**Reply by Welding-Torch** [reply]
Hi, I created PR #463 for this.
